service/controller/clusterapi/v29/resource/tccp/template_render_test.go

```diff
@@ -160,7 +161,23 @@ func defaultControllerContext() controllercontext.Context {
 				},
 			},
 			TenantCluster: controllercontext.ContextStatusTenantCluster{
-				AvailabilityZones:     []string{"eu-central-1a", "eu-central-1b", "eu-central-1c"},
+				AvailabilityZones: []controllercontext.ContextStatusTenantClusterAvailabilityZone{
+					{
+						Name:          "eu-central-1a",
+						PrivateSubnet: mustParseCIDR("10.100.3.0/27"),
+						PublicSubnet:  mustParseCIDR("10.100.3.32/27"),
```

Comment:
These don't take effect on rendering outcome yet. That will be done in separate PR.